src/main/java/com/image/gallery/search/util/TokenUtil.java

```diff
@@ -0,0 +1,66 @@
+package com.image.gallery.search.util;
+
+import com.image.gallery.search.exseption.GetDataFromUrlException;
+import com.image.gallery.search.exseption.InvalidTokenResponseException;
+import com.image.gallery.search.exseption.MyEncodingException;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpHeaders;
+import org.apache.http.StatusLine;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.json.JSONObject;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TokenUtil {
+    private static final String IMAGE_URL = "http://interview.agileengine.com/images";
+
+    public String getToken() {
```

Comment:
it's better to improve naming here as well is it security / access / secret token etc ?